pyproject.toml

```diff
@@ -50,7 +50,6 @@ dependencies = [
     'jsonschema==3.2.0',
     'kubernetes==24.2.0',
     'mac-vendor-lookup==0.1.11',
-    'matplotlib<=3.5.1',
```

Comment:
Doesn't this imply that NetworkDrawing.py be removed? (or never, ever called)?

Reply:
This was causing the builds to fail for some reason after the switch to pyrpoject toml.  Prior to that change, matplotlib wasn't even in the `requires` section.  Is it pulled in elsewhere in the dep chain perhaps?

Reply:
NVM, it was a missing build dependency.